traffic_ops/testing/api_contract/v4/conftest.py

```diff
@@ -311,6 +312,23 @@ def to_login(to_args: ArgsType) -> TOSession:
 	return to_session
 
 
+@pytest.fixture(scope="session", name="db_connection")
+def open_db_connection():
+	"""
+	Creates new traffic ops db connection.
+	:returns: New Traffic ops database connection
+	"""
+	conn = psycopg2.connect(
+            user="traffic_ops",
+            password="twelve",
+            host="127.0.0.1",
+            port=5432,
+            database="traffic_ops",
+            sslmode="disable"
```

Comment:
this DB setup will only work for a local install or an install with that specific username and password, this should be set in our config file instead

Reply:
Changed it!